`npm run format` does not run properly

```
​C:\Users\crutchcorn\git\GitGui\isomorphic-git [renameBranch|REBASE-i]
λ  npm run format

> isomorphic-git@0.0.0-development format C:\Users\crutchcorn\git\GitGui\isomorphic-git
> nps format

Scripts must resolve to strings. There is no script that can be resolved from "format" https://github.com/sezna/nps/blob/v5.9.8/other/ERRORS_AND_WARNINGS.md#missing-script
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! isomorphic-git@0.0.0-development format: `nps format`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the isomorphic-git@0.0.0-development format script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\crutchcorn\AppData\Roaming\npm-cache\_logs\2020-07-02T03_17_55_929Z-debug.log
```

--- a/package-scripts.js
+++ b/package-scripts.js
@@ -33,6 +33,9 @@ module.exports = {
       default: `eslint .`,
       fix: optional(`eslint --fix .`),
     },
+    format: {
+      default: series.nps('lint.fix'),
+    },
     watch: {
       default: concurrent.nps('watch.rollup', 'watch.jest'),
       rollup: runInNewWindow('rollup -cw'),

